src/NLog.Targets.AppCenter/AppCenterTarget.cs

```diff
@@ -192,10 +200,72 @@ private void TrackEvent(string eventName, Exception exception, IDictionary<strin
                 properties = properties ?? new Dictionary<string, string>(1);
                 if (properties.Count < 20)
                     properties["EventName"] = eventName;
-                Microsoft.AppCenter.Crashes.Crashes.TrackError(exception, properties);
+
+                Microsoft.AppCenter.Crashes.Crashes.TrackError(exception, properties, attachmentLogs);
             }
 
             Microsoft.AppCenter.Analytics.Analytics.TrackEvent(eventName, properties);
         }
+
+        /// <remarks>
+        ///     The maximun file size (not compressed) is 10mb
+        ///     The maximum attached file (compressed) size is 1mb.
+        ///     The maximum attached file count is 10.
+        /// </remarks>
+        private Microsoft.AppCenter.Crashes.ErrorAttachmentLog[] BuildAttachmentLogs(LogEventInfo logEvent)
+        {
+            var path = RenderLogEvent(PathToCrashAttachmentDirectory, LogEventInfo.CreateNullEvent());

```

Comment:
Please change `LogEventInfo.CreateNullEvent()` to use the input-parameter `logEvent`